src/components/common/LangDropdown.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import lang from "../images/lang.png";
+import "../index.css";
+
+function LangDropdown() {
+  const { t, i18n } = useTranslation();
+
+  const handleChange = (e) => {
+    switch (e.target.innerText) {
+      case "Turkish":
+      case "Türkçe":
+      case "التركية":
+        i18n.changeLanguage("tr");
+        return;
+      case "Arabic":
+      case "Arapça":
+      case "العربية":
+        i18n.changeLanguage("ar");
+        return;
+      case "English":
+      case "İngilizce":
+      case "الإنجليزية":
+        i18n.changeLanguage("en");
+        return;
+      default:
+        return;
+    }
+  };
+
+  return (
+    <div className="langDropdown">
+      <img className="lang" alt="lang. selector" src={lang} />
+      <ul className="lang">
+        <li className="lang" onClick={(e) => handleChange(e)}>
+          {t("lang.en")}
+        </li>
+        <li className="lang" onClick={(e) => handleChange(e)}>
```

Comment:
the way you write this code if it doesn't have a value for e is to instead use currying (google to review currying if needed)

the cleanest way is as so
1. define an enum (fake in js but w/e) for languages
https://stackoverflow.com/questions/287903/what-is-the-preferred-syntax-for-defining-enums-in-javascript

2. pass it in as the click handler

3. switch-case in handlelanguageclick
```
const handleLanguageClick = (language) => (e) => {
};
```